src/main/java/seedu/nutrigoals/commons/core/Messages.java

```diff
@@ -9,5 +9,6 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_MEAL_DISPLAYED_INDEX = "The meal index provided is invalid";
     public static final String MESSAGE_MEALS_LISTED_OVERVIEW = "%1$d meals listed!";
+    public static final String MESSAGE_MULTIPLE_TAGS_ERROR = "Each tag should only be used once!";
```

Comment:
Maybe it'll be clearer this way?
```suggestion
    public static final String MESSAGE_REPEATED_PREFIX_ERROR = "Each prefix should only be used at most once!";
```

Currently, `add n/sushi n/bake rice c/1000 t/dinner` results in this error message: `Each tag should only be used once!` But tags could just refer to the dinner tag: `t\dinner`.

Reply:
oya ok i shall go change this